src/DI/WebpackExtension.php

```diff
@@ -73,9 +79,7 @@ public function loadConfiguration()
 				new Statement(Client::class)
 			]);
 
-		$assetResolver = $builder->addDefinition($this->prefix('assetResolver'))
-			->setClass(AssetResolver\AssetResolverInterface::class)
-			->setFactory($config['assetResolver']);
+		$assetResolver = $this->setupAssetResolver($config);
 
 		if ($this->debugMode && $config['debugger']) {
```

Comment:
> shouldnt this be one thing?

Not necessarily, but having `$this->debugMode` in there can have unintended effects, so I removed it (af5af4f), thanks for pointing out :)